src/neuigkeiten/2022-01-12-release/index.md

```diff
@@ -0,0 +1,74 @@
+---
+draft: false
+layout: layout-neuigkeiten
+date: 2022-01-12
+title: Neue Feiertagsregelungen, Abwesenheitsanträge an Feiertagen und ein neues Kalenderdesign
+author: Tobias Schneider
+release-version: 4.28.0
+teaserImage: /static/update.jpg
+---
+
+_Feiertagsregelung für Österreich, die Schweiz und das Vereinigtes Königreich, Abwesenheiten an Feiertagen und ein neues Kalenderdesign_
+
+<!-- more -->
+
+## Feiertagsregelung
+
+Ab sofort kann die Feiertagsregelung, neben den Bundesländern für Deutschland, auch mit den österreichischen Bundesländern,
+den Schweizer Kantonen sowie mit den Landesteilen des Vereinigtes Königreiches konfiguriert werden. Diese Einstellung kann, wie gewohnt,
+global in den Einstellungen der Urlaubsverwaltung konfiguriert als auch im Konto einer Person überschrieben werden.
+Der Name der Konfiguration hat sich von 'Bundesland' zu 'Feiertagsregelung' geändert.
+
+<div class="flex gap-4 flex-col md:flex-row">
+    <picture>
+        <source srcset="public_holiday_person.avif" type="image/avif" />
+        <source srcset="public_holiday_person.webp" type="image/webp" />
+        <img
+          src="public_holiday_person.png"
+          alt="Personübersicht mit Feiertagregelungauswahl"
+          decoding="async"
+          loading="lazy"
+        />
+    </picture>
+    <picture>
+        <source srcset="public_holiday_person_change.avif" type="image/avif" />
+        <source srcset="public_holiday_person_change.webp" type="image/webp" />
+        <img
+          src="public_holiday_person_change.png"
+          alt="Feiertagregelungauswahl einer Person"
+          decoding="async"
+        />
+    </picture>
+</div>
+
+## Abwesenheiten an Feiertagen
+
+Die [Feiertagsregelung](#feiertagsregelung) wurde zudem um den Eintrag _Keine Feiertagsregelung_ erweitert. Mit dieser Einstellung
+ist es für die gesamte Urlaubsverwaltung sowie für bestimmte Personen möglich die Feiertagsregelung außer Kraft zu setzen und
+Abwesenheiten auch an Feiertagen einzutragen.
```

Comment:
Vielleicht könnte man hier noch einen Anwendungsfall beschreiben?

Reply:
Bin mir gerade nicht sicher ob das von Vor- oder Nachteil ist, sodass wir dann die Benutzer in eine Richtung "drücken". Würde auch gerne sehen wie einfallsreich sie damit umgehen...hmhmh.